libavutil/hwcontext_vaapi.c

```diff
@@ -992,7 +992,7 @@ static const struct {
 } vaapi_drm_format_map[] = {
 #ifdef DRM_FORMAT_R8
     DRM_MAP(NV12, 2, DRM_FORMAT_R8,  DRM_FORMAT_RG88),
-    DRM_MAP(NV12, 2, DRM_FORMAT_R8,  DRM_FORMAT_GR88),
+    DRM_MAP(NV21, 2, DRM_FORMAT_R8,  DRM_FORMAT_GR88),
```

Comment:
Mapping UV plane of NV12 to DRM_FORMAT_GR88 is WA, and I think we needn't add support for NV21 and  should remove `DRM_MAP(NV12, 2, DRM_FORMAT_R8,  DRM_FORMAT_GR88)` after applying the first commit.

BTW so far we don't have a driver to support NV21

Reply:
Ok, I will delete this line.